composer.json

```diff
@@ -16,14 +16,15 @@
     ],
     "require": {
         "php": ">=7.3",
-        "symfony/process": "~3.0|^4.0|^5.0"
+        "symfony/process": "^4.4|^5.0|^6.0"
     },
     "suggest": {
         "symfony/filesystem": "Allows using Symfony Filesystem"
     },
     "require-dev": {
         "roave/security-advisories": "dev-master",
-        "phpunit/phpunit": "^9.0"
+        "symfony/filesystem": "^4.4|^5.0|^6.0",
```

Comment:
What's the reason for adding this? It's already in the suggest and I don't think we have any tests that require it at the minute

Reply:
Is is easier for dev to have "symfony/filesystem": "^4.4|^5.0|^6.0", installed and check if it is correct by IDE or static code analyzer.